addhost.php

```diff
@@ -16,7 +16,7 @@
 $init_modules = array();
 require __DIR__ . '/includes/init.php';
 
-$options = getopt('g:p:f::');
+$options = getopt('Pg:p:f::');
```

Comment:
Not sure we should use upper case P here. Maybe i = icmp?

Reply:
I chose -P here and in snmp-scan.py to get the same letter. -i is already used in snmp-scan.py, and -p is used here in addhost.php. I can still change it to -p and -i if you'd prefer.

Reply:
Maybe leave it then. Just easy to confuse people that's all.